docs/release-notes/master.adoc

```diff
@@ -27,41 +36,14 @@ include::topics/ref_fixed-issues-7-3-0.adoc[leveloffset=+2]
 
 include::topics/ref_known-issues-7-3-0.adoc[leveloffset=+2]
 
-
-[id="mta-7-2-2"]
-== {ProductShortName} 7.2.2
-
-include::topics/ref_resolved-issues-7-2-2.adoc[leveloffset=+2]
-
-include::topics/ref_known-issues-7-2-2.adoc[leveloffset=+2]
-
-[id="mta-7-2-1"]
-== {ProductShortName} 7.2.1
-
-include::topics/ref_resolved-issues-7-2-1.adoc[leveloffset=+2]
-
-include::topics/ref_known-issues-7-2-1.adoc[leveloffset=+2]
-
-
-[id="mta-7-2-0"]
-== {ProductShortName} 7.2.0
-
-include::topics/ref_new-features-7-2-0.adoc[leveloffset=+2]
-
-// include::topics/ref_upgrade-notes-7-2-0.adoc[leveloffset=+2]
-
-include::topics/ref_resolved-issues-7-2-0.adoc[leveloffset=+2]
-
-include::topics/ref_known-issues-7-2-0.adoc[leveloffset=+2]
-
 //snippets required for release notes and other locations
 include::topics/analysis-dependency-rule-snippet.adoc[]
 
 include::topics/analyzer-rbac-known-issue-snippet.adoc[]
 
 // include::topics/svn-checkout-fails-issue-snippet.adoc[]
 
-For a complete list of all known issues, see the list of link:https://issues.redhat.com/issues/?filter=12447880[{ProductShortName} 7.2.0 known issues] in Jira.
+For a complete list of all known issues, see the list of link:https://issues.redhat.com/issues/?filter=12469170[{ProductShortName} 7.3.1 known issues] in Jira.
```

Comment:
This filter lists all the bugs fixed in MTA 7.3.1. Was that the intention?
Maybe I am getting it wrong, but for me known issues sound more like bugs that are not yet fixed.